docs/advanced/secure-setup.md

```diff
@@ -0,0 +1,314 @@
+# <img class="dcr-icon" src="/img/dcr-icons/LockEye.svg" /> Secure Setup
+
+Last updated for CLI release v{{ cliversion }}.
+
+---
+
+This document will guide how to setup a two system method for holding your Decred securely.
+Some of the commands given will be specific to Debian based systems, but the principles/tools can be applied to almost any OS.
+
+**Please note that this setup requires two computers.**
+
+Although focused on Decred and Raspberry Pi, this guide can also be used for
+more general hardware devices and almost any cryptocurrency.
+
+---
+
+## Scope and Limitations
+
+The scope of this guide is to secure your cryptocurrency holdings from common malware and light threats.
+If you are a normal cryptocurrency holder and follow this guide exactly, it should give a good level of security.
+
+The guide assumes some general technical knowledge and that you can fix any hitches you come across.
+
+**This setup will not protect you against a state-level attacker or a persistent threat.**
+
+[General Security](general-security.md) offers more techniques to keep your system secure.
+
+---
+
+## The Setup
+
+This will be a multi-system setup.
+
+![Network diagram](/img/secure-setup.png)
+
+* A laptop or desktop that will be running the dcrd node, referred to as `Computer-A`.
+* A secure system that will be running the wallet, referred as `Wallet-B`.
+  This guide will use a Raspberry Pi 4, however it could also be a laptop or desktop.
+  This system should be connected to an external monitor and keyboard, and it
+  should also connect to the local network via ethernet.
+* All details/commands that come within curly braces `{}` cannot be copy-pasted,
+  you will need to remove the curly braces and edit them to suit your setup.
+
+### Benefits of this Setup
+
+* `Wallet-B` is kept offline and turned off most of the time.
+  This greatly reduces attack surface/opportunity.
+* An attacker would have to compromise dcrd on `Computer-A` first, and then find a
+  way to jump into `Wallet-B` using the RPC to compromise and exfiltrate data.
+  This is highly unlikely and difficult to pull off.
+* dcrd on `Computer-A` can be used for other purposes in your local network (eg. other wallets, DEX, etc),
+  so you don't need to maintain multiple dcrd instances within your local network.
+
```

Comment:
Before jumping straight into the instructions, I think it would be beneficial to add two more subsections here with a bit more preliminary explanation:

## Hardware

### Computer A - The Node

In general, you can use almost any computer to run the node. See the [Minimum Recommended Specifications](https://github.com/decred/dcrd#minimum-recommended-specifications-dcrd-only) for dcrd. `Computer A` is expected to remain online most of the time, but it's not required. For a budget device with a low energy requirement, it’s fine to use a Raspberry Pi with at least 2 GB memory and 16 GB disk space. It would also be fine to use a laptop or desktop. This guide assumes users are aware that, since Computer A is connecting to the public internet, then it is necessarily vulnerable to attacks.

### Wallet B

Since the wallet needs to have the utmost security in this setup, it’s important to have a minimal attack surface. This guide suggests that this wallet be a cold wallet: powered off most of the time, because it would be nearly impossible to attack a system which is powered off. This guide suggests using a Raspberry Pi for the wallet, although other hardware can be used.

### Finally, this guide suggests using a mouse, keyboard, and monitor to access 

## Operating Systems

Since `Computer A` is going to be the more vulnerable device, it doesn’t really matter which operating system you choose. `dcrd` can be installed on anything from Windows to Mac to Linux to OpenBSD. As mentioned before, you can just just use a laptop or desktop. However, if you plan to use a budget device like a Raspberry Pi that you can leave online, consider using Linux or OpenBSD, which will reduce your attack surface.

This guide assumes you already have an operating system on `Computer A`, and won’t describe how to install a new one.

`Wallet B` will be the more secure device. Use an operating system you feel comfortable with. Consider using Linux or OpenBSD to reduce your attack surface. Raspbian works fine if you want a GUI, and Ubuntu Desktop is another good choice for beginners who don’t need a GUI.

Again, this guide will give commands specific to Debian/Linux based systems. All details/commands that come within curly braces `{}` cannot be copy-pasted. You will need to remove the curly braces and edit them to suit your setup.

Reply:
> Since Computer A is going to be the more vulnerable device, it doesn’t really matter which operating system you choose. 

I don't follow the logic here. What do you think of something like

> Since Computer A is not storing any sensitive information, the choice of operating system is less important than the choice for Wallet B.

Reply:
Yep, good change.